doc/src/docs/EngineeringReference/13f-EncyclopaedicRefs.md

```diff
@@ -544,16 +544,12 @@ EvapHeatLossRegComf = 0.42\*(IntHeatProd - ActLevelConv)
 
 EvapHeatLossRegComf = 0.0
 
-EvapHeatLossDiff = 0.4148\*(SkinComfVpress - VapPress)
+EvapHeatLossDiff = 0.00305\*(5733.0 - 6.99\*IntHeatProd - VapPress)
 
 EvapHeatLoss = EvapHeatLossRegComf + EvapHeatLossDiff
 
 Where,
 
-0.68 is the passive water vapor diffusion rate, (g/h·m<sup>2</sup>·Torr)
-
-0.61 is the latent heat of water, (W·h/g)
-
```

Comment:
I see nothing else wrong between docs and code except for the following:

The remaining definitions below the "Where:" at this point are also not used and should be deleted (i.e., Psk, Psk= 1.92..., SatSkinVapPress = ..., Tskr = ..., and SkinTempComf), as should the "Where:". I also question the following paragraph. Does the model still calculate skin temperature? I don't see it except in comment lines. I also wonder if the new model still relies on thermal sensation votes from subjects at KSU and Denmark? Revise the paragraph wording to simply state the obvious? If you agree, then the entries in Table 81 that were deleted should also be removed (probably harder to do than it should be).

I could delete the "Where:" block and revise the following paragraph, but I an not familiar enough with the recent changes so I ask these questions instead.